app/Models/Language.php

```
@@ -293,9 +293,15 @@ public function create(array $data): int
 
-        // If this is set as default, unset other defaults
+        // Invalidate ONCE, and only after the languages list is in its final
+        // consistent state. When this row becomes the default, setDefault()
+        // flips the other rows and invalidates at its own commit — invalidating
+        // here first would let a concurrent read re-cache the intermediate
+        // multiple-defaults state for the full TTL.
         if ($isDefault) {
             $this->setDefault($code);
+        } else {
+            \App\Support\QueryCache::delete('i18n_languages');
         }
```

PHPMD (2.15.0)

[error] The method create uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)